### ISSUE TITLE
Loop confirmation target input validation

### ISSUE THREAD

**Issue by pfergi42** [lightninglabs/lightning-terminal#165]
The placeholder text for confirmation target says "Example: 6" but the error says the number must be between 100 & 400.

<img width="904" alt="Screen Shot 2020-12-27 at 11 37 42 AM" src="https://user-images.githubusercontent.com/4021240/103178511-45b83280-4838-11eb-8756-a98684439900.png">

<img width="273" alt="Screen Shot 2020-12-27 at 11 37 52 AM" src="https://user-images.githubusercontent.com/4021240/103178514-4781f600-4838-11eb-8f1a-fde5c5a171ae.png">

<img width="394" alt="Screen Shot 2020-12-27 at 11 37 58 AM" src="https://user-images.githubusercontent.com/4021240/103178515-4781f600-4838-11eb-8db4-78b815191889.png">

**Comment by jamaljsr** [comment]
Good point. I'd agree the placeholder should display a more realistic value. Thanks for the feedback.

**Comment by clashicly** [comment]
This is still an issue with Lightning Terminal v0.5.0-alpha shipped with Umbrel 0.4.6.

Steps to reproduce:
If Additional Options are clicked with an integer value in Confirmation Target field less than 100, a validation error with "Confirmation target must be between 100 and 400." is displayed and there is no ability to move to step 2.

If Confirmation Target is ignored, but Destination is entered, you can move to step 2.

If Additional Options are not used, you can move to Step 2 with Next button and successfully perform a Loop Out.

**Comment by jsmalaga** [comment]
Hi,
I did purchase a channel and I am getting inbound traffic via your node "Zerofeerouting" and outbound traffic via another channel i have open "ln.nicehash.com".
This is causing me a problem.
The Local liquidity in my channel "ln.nicehash.com" drain very fast and the routing/Forwarding stops!

I am trying to rebalance "Loop in" but with no success.

Is there any way to config and force the routing in/out via the same node "Zerofeerouting" ?
How could auto rebalance my outbound node "ln.nicehash.com" to keep liquidity high?
Thanks,

**Comment by guggero** [comment]
@jsmalaga please don't spam existing issues with your unrelated problem!

**Comment by cypcz** [comment]
This is still the case. Why is minimum a 100?

**Comment by gStart9** [comment]
4 years later... this is still the case.  Why is the minimum 100?
6 is given as an example, but apparently not valid.  Leaving it blank makes step 2 available.  But what is the default?
